mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -107,12 +109,14 @@ CLASS avr net/minecraft/entity/player/PlayerEntity
 	METHOD dI getSpawnPosition ()Lev;
 	METHOD dJ isSpawnForced ()Z
 	METHOD dK getEnchantmentTableSeed ()I
+	METHOD dL getNextLevelExperience ()I
 	METHOD dM getHungerManager ()Laxr;
 	METHOD dN canFoodHeal ()Z
 	METHOD dO canModifyWorld ()Z
 	METHOD dP getEnderChestInventory ()Laza;
 	METHOD dQ dropShoulderEntities ()V
 	METHOD dR getScoreboard ()Lcso;
+	METHOD dS getNameAndUuid ()Ljm;
```

Comment:
This is really a display name for an entity in the command output that allows you to copy the player name, right?

Reply:
No, this is a "PlayerName (123e4567-e89b-12d3-a456-426655440000)" string

Reply:
Ah, nice